packages/manager/src/features/linodes/LinodesCreate/SelectPlanPanel.tsx

```diff
@@ -410,6 +414,24 @@ export const SelectPlanPanel: React.FC<CombinedProps> = (props) => {
 
     const shared = [...nanodes, ...standards];
 
+    if (!isEmpty(proDedicated)) {
+      tabs.push({
+        render: () => {
+          return (
+            <>
+              <Typography data-qa-prodedi className={classes.copy}>
+                Pro Dedicated CPU instances are for very large plans. They only
+                have AMD 2nd generation processors or above.
+              </Typography>
```

Comment:
Is this copy temporary? It does not sound right to me. I left what I think sounds better below.

```suggestion
              <Typography data-qa-prodedi className={classes.copy}>
                Pro Dedicated CPU instances are for very demanding workloads. They
                have AMD Epyc 2nd generation processors or newer.
              </Typography>
```

Reply:
Not sure if `AMD Epyc` has been finalized so just going to leave it as `AMD 2nd generation` for now